src/SFA.DAS.AANHub.Api/Common/ActionResponseControllerBase.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.AANHub.Application.Mediatr.Common;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+
+namespace SFA.DAS.AANHub.Api.Common
+{
+    public class ActionResponseControllerBase : ControllerBase
+    {
+        private readonly ILogger<ActionResponseControllerBase> _logger;
+
+        public ActionResponseControllerBase(ILogger<ActionResponseControllerBase> logger) => _logger = logger;
+
+        protected IActionResult GetResponse<T>(ValidatedResponse<T> response, BaseRequestDetails requestDetails) where T : class
+        {
+            if (response.Result == null && response.Errors.Count == 0)
```

Comment:
Use IsValidResponse instead of errors count

Reply:
This has been updated now